api/lib/config/environments/preview.config.ts

```diff
@@ -1,18 +1,22 @@
 export default {
+    logging: 'debug',
+    secureCookie: false,
     services: {
-        ccdDataApi: 'https://ccd-data-store-api-aat.service.core-compute-aat.internal',
-        ccdDefApi: 'https://ccd-definition-store-api-aat.service.core-compute-aat.internal',
+        ccd: {
+            componentApi: 'https://ccd-api-gateway-web-aat.service.core-compute-aat.internal',
+            dataApi: 'https://ccd-data-store-api-aat.service.core-compute-aat.internal',
+        },
```

Comment:
If we look at the other configs/services we're going with a flatter structure no need to group them. Also I don't think we need coh and documents ?

Reply:
cool will change